android/src/main/kotlin/com/muljin/zendesk/ZendeskHelper.kt

```diff
@@ -193,4 +197,22 @@ class ZendeskHelper : FlutterPlugin, MethodCallHandler, ActivityAware {
                 .show(activity, chatConfiguration)
 
     }
+
+    private fun registerPushToken(call: MethodCall, flutterResult: Result) {
+        val pushToken = call.argument<String>("pushToken")
+        if(pushToken == null) {
+            flutterResult.error("registerPushToken", "pushToken is required", null)
+            return
+        }
+
+        val pushProvider = Chat.INSTANCE.providers()?.pushNotificationsProvider()
+
+        if(pushProvider == null) {
+            flutterResult.error("registerPushToken", "pushProvider is null", null)
+            return
+        }
+
+        pushProvider.registerPushToken(pushToken)
+        flutterResult.success(true)
+    }
```

Comment:
Não precisa fazer isso se já tiver testado e tiver tudo certo:

Não seria melhor colocarmos isso no initialize? pois assim poderiamos ter um String? como parâmetro e caso for não nulo poderíamos usar essa funcionalidade